src/client/const.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+    API_BASE_URL: 'http://localhost:3030/users',
```

Comment:
i tend to prefer to just do:

```js
export const API_BASE_URL: 'http://localhost:3030/users'
```

because then your imports are easier to read:

```js
import { API_BASE_URL } from ...

```

instead of

```js
import consts from ....

// then using as
consts.API_BASE_URL
```